src/building-blocks/FormFieldSet.tsx

```diff
@@ -30,10 +30,19 @@ class FormFieldSet extends Component<IFormFieldSetProps> {
     return fillInFieldSet(this.props.fieldSet);
   }
 
+  private get filteredFieldConfigs () {
+    const fieldConfigs = getFieldSetFields(this.fieldSet);
+
+    if (!fieldConfigs.some(fc => !!fc.insertIf)) {
```

Comment:
may as well use the standard `fieldConfig` variable name

Reply:
Fair.